### DISCUSSION TITLE
1.9.2

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/LSPosed#2810] [New Release]
### Changelog
- Fix `IContentProvider` for Android P and below
- Properly handle invalid `Parcel`
- Fix invalid `setOverrideTable` address on some devices
- Move manager APK to the module path
- Fix remote preferences listener not working on system_server
- Fix hook crash on Android 14
- Fix static method hook on some Android 14 devices
- Fix some `LoadedApk`s not calling `onPackageLoad`
- Fix the wrong path of the new `XSharedPreferences`
- Add tips about deploying optimizations in Android Studio

### 更新日志
- 修复 Android P 及以下版本的 `IContentProvider`
- 正确处理无效的 `Parcel`
- 修复某些设备上无效的 `setOverrideTable` 地址
- 将管理器 APK 移动到模块路径
- 修复系统框架上 remote preference 监听器无效的问题
- 修复 Android 14 上的 hook 错误
- 修复某些设备上 Android 14 的静态方法 hook 问题
- 修复一些 `LoadedApk` 没有回调 `onPackageLoad`
- 修复新 `XSharedPreferences` 的路径错误
- 增加 Android Studio 部署优化提示

### Contributors
@rawer886 @Howard20181 @yujincheng08 @Dr-TSNG @naicfeng @vvb2060 

**Full Changelog**: https://github.com/LSPosed/LSPosed/compare/v1.9.1...v1.9.2

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.9.2'>1.9.2</a>.</em>

**Comment by SalamiSimon** [comment]
Auto updater tried installing the Riru version when on Zygisk and failed. Easy to just flash from GitHub but just a heads up.

**Reply by faraoncik13** [reply]
![Screenshot_2023-12-02-17-04-22-389_com miui bugreport](https://github.com/LSPosed/LSPosed/assets/84202708/78184373-c5de-41f0-bc68-c48a5f1526f3)
Doesn't work in version Hyper OS 23.11.27

**Comment by qingsonglove** [comment]
我的安装的是win10 安卓子系统WSA，是已经root过的。Magisk版本是 d4e552d0(26301)Canary，从Magisk管理器上的"从本地安装" 去安装 Shamiko0.7.4-188-release.zip和LSPosed1.9.2-7024zygisk-release.zip后，重启安卓子系统之后，在Magisk管理器界面上的模块一栏中，没显示 Shamiko和LSPosed模块。请问这是怎么回事，应该如何解决？
![0](https://github.com/LSPosed/LSPosed/assets/149339462/d7c1cac1-9aa9-4600-a9e2-592cfdbfbbed)
![1](https://github.com/LSPosed/LSPosed/assets/149339462/5ed8a1fc-3234-4567-9d4d-17ff859dadb4)
![2](https://github.com/LSPosed/LSPosed/assets/149339462/1b73afc1-0ec3-49ff-9470-5d33c5e30c9b)

**Comment by qingsonglove** [comment]
我的安装的是Win10 安卓子系统WSA，是已经root过的。Magisk版本是 d4e552d0(26301)Canary，从Magisk管理器上的"从本地安装" 去安装 Shamiko0.7.4-188-release.zip和LSPosed1.9.2-7024zygisk-release.zip后，重启安卓子系统之后，在Magisk管理器界面上的模块一栏中，没显示 Shamiko和LSPosed模块。请问：1、为什么会出现这种情况？2、应该如何解决？
![Uploading 2.JPG…]()

**Comment by shangshanlu** [comment]
The installation in WSA was unsuccessful.Previous versions can be installed.

**Comment by faraoncik13** [comment]
> ### Changelog
> * Fix `IContentProvider` for Android P and below
> * Properly handle invalid `Parcel`
> * Fix invalid `setOverrideTable` address on some devices
> * Move manager APK to the module path
> * Fix remote preferences listener not working on system_server
> * Fix hook crash on Android 14
> * Fix static method hook on some Android 14 devices
> * Fix some `LoadedApk`s not calling `onPackageLoad`
> * Fix the wrong path of the new `XSharedPreferences`
> * Add tips about deploying optimizations in Android Studio
> 
> ### 更新日志
> * 修复 Android P 及以下版本的 `IContentProvider`
> * 正确处理无效的 `Parcel`
> * 修复某些设备上无效的 `setOverrideTable` 地址
> * 将管理器 APK 移动到模块路径
> * 修复系统框架上 remote preference 监听器无效的问题
> * 修复 Android 14 上的 hook 错误
> * 修复某些设备上 Android 14 的静态方法 hook 问题
> * 修复一些 `LoadedApk` 没有回调 `onPackageLoad`
> * 修复新 `XSharedPreferences` 的路径错误
> * 增加 Android Studio 部署优化提示
> 
> ### Contributors
> @rawer886 @Howard20181 @yujincheng08 @Dr-TSNG @naicfeng @vvb2060
> 
> **Full Changelog**: [v1.9.1...v1.9.2](https://github.com/LSPosed/LSPosed/compare/v1.9.1...v1.9.2)
> 
> _This discussion was created from the release [1.9.2](https://github.com/LSPosed/LSPosed/releases/tag/v1.9.2)._

Doesn't work in version 23.11.27 Hyper OS. Correct the error please.
![Screenshot_2023-12-02-17-04-22-389_com miui bugreport](https://github.com/LSPosed/LSPosed/assets/84202708/65427911-e0b9-496e-960b-ccd109ed9fcd)

**Comment by mick4y** [comment]
This Lsposed is not longer working hope update the shell

**Comment by Miracle-66** [comment]
May I kindly ask how I can join the beta testing group?

**Comment by Denis-666** [comment]
报错:'Zygish 未启动,此模块暂停加载'

**Comment by tki7yu** [comment]
内测真难进啊
